bitcoin/src/blockdata/locktime/absolute.rs

```diff
@@ -132,6 +132,10 @@ impl LockTime {
     /// assert!(LockTime::from_height(741521).is_ok());
     /// assert!(LockTime::from_height(1653195600).is_err());
     /// ```
+    /// 
+    /// # Error
+    /// 
+    /// If `n` does not represent a block height value (see documentation on [`LockTime`]).
```

Comment:
> If `n` is above the limit.

Would convey the meaning better because this is not `from_consensus`.

Reply:
sure will change it